collection-scripts/gather

```diff
@@ -38,5 +38,8 @@ done
 # Collect System Audit Logs
 /usr/bin/gather_audit_logs
 
+# Collect host files
+/usr/bin/gather_host_files
```

Comment:
If were are going to gather files from 'every node' then we shouldn't to this by default - some clusters have LARGE node sets, and this could cause must-gather to run for an indefinite amount of time.